README.md

```diff
@@ -14,6 +15,22 @@ Sinatra application that provides a preview of how the OAS documents will be ren
 
 ## Installation and Usage
 
+### Using Docker
+
+You can run using Docker and it will serve the current directory (this will usually be the api-specification repo):
+
+```bash
+docker run --rm -p 4567:4567 -v `pwd`:/definitions -e 'OAS_PATH=/definitions' mheap/nexmo-oas-renderer:latest
```

Comment:
I think we should use a placeholder for `mheap/nexmo-oas-renderer:latest`, WDYT?